static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -46,11 +46,15 @@ define([
                 description: 'Sign up and we\'ll send you the Guardian US Campaign Minute, once per day.',
                 successHeadline: 'Thank you for signing up to the Guardian US Campaign minute',
                 successDescription: 'We will send you the biggest political story lines of the day',
-                modClass: 'post-article',
+                modClass: config.page.isMinuteArticle ? 'post-article' : 'end-article',
                 insertMethod: function () {
-                    return function ($iframeEl) {
-                        $iframeEl.insertAfter('.js-article__container');
-                    };
+                    if (config.page.isMinuteArticle) {
+                        return function ($iframeEl) {
+                            $iframeEl.insertAfter('.js-article__container');
+                        };
+                    } else {
+                        return insertBottomOfArticle();
```

Comment:
Nope because the call is `insertMethod()($iframeEl);` so on the others they call `insertBottomOfArticle()` as the first execution which will immediately return the function whereas in this one, because there is a function assigned to the property the first execution calls that function containing the if statement which returns either that function in the `if` or calls `insertBottomOfArticle()` which returns the same function as the other calls. There's that extra level so we can do the if check.

I mean, it's a bit of a head****. :/